structure_tests/drivers/docker_driver.go

```diff
@@ -0,0 +1,341 @@
+// Copyright 2017 Google Inc. All rights reserved.
+
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+
+//     http://www.apache.org/licenses/LICENSE-2.0
+
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package drivers
+
+import (
+	"bytes"
+	"fmt"
+	"io/ioutil"
+	"os"
+	"strings"
+	"testing"
+	"time"
+
+	"context"
+
+	"bufio"
+
+	"regexp"
+
+	"github.com/GoogleCloudPlatform/runtimes-common/structure_tests/types/unversioned"
+	"github.com/fsouza/go-dockerclient"
+)
+
+type DockerDriver struct {
+	cli docker.Client
+}
+
+var originalImage, currentImage string
```

Comment:
Can these be on the struct instead? It's a bit scary to have them at the package level.